AWS/aws-cloudfront/_meta/smart-descriptions.json

```diff
@@ -0,0 +1,151 @@
+[
+  {
+    "value": "{source.ip} called domain {destination.domain} and get the response from the nearest edge cache in {server.geo.name}",
+    "conditions": [
+      {
+        "field": "event.action",
+        "value": "Hit"
+      },
+      {
+        "field": "source.ip"
+      },
+      {
+        "field": "destination.domain"
+      },
+      {
+        "field": "server.geo.name"
+      }
+    ]
+  },
+  {
+    "value": "{source.ip} called domain {destination.domain}, then the edge in {server.geo.name} found that cache expired, so he contact the origin server",
+    "conditions": [
+      {
+        "field": "event.action",
+        "value": "RefreshHit"
+      },
+      {
+        "field": "source.ip"
+      },
+      {
+        "field": "destination.domain"
+      },
+      {
+        "field": "server.geo.name"
+      }
+    ]
+  },
+  {
+    "value": "{source.ip} called domain {destination.domain}, then edge server in {server.geo.name} can't found the object in the cache, so he contact the origin server",
+    "conditions": [
+      {
+        "field": "event.action",
+        "value": "Miss"
+      },
+      {
+        "field": "source.ip"
+      },
+      {
+        "field": "destination.domain"
+      },
+      {
+        "field": "server.geo.name"
+      }
+    ]
+  },
+  {
+    "value": "{source.ip} called domain {destination.domain}, but the request was denied because of CloudFront quota",
+    "conditions": [
+      {
+        "field": "event.action",
+        "value": "LimitExceeded"
+      },
+      {
+        "field": "source.ip"
+      },
+      {
+        "field": "destination.domain"
+      }
+    ]
+  },
```

Comment:
```suggestion
  {
    "value": "Cannot deliver resource {url.path} to {source.ip} because of the CloudFront quota was exceeded",
    "conditions": [
      {
        "field": "event.action",
        "value": "LimitExceeded"
      },
      {
        "field": "source.ip"
      },
      {
        "field": "url.path"
      }
    ]
  },
```